encoding/protobuf/v2/error_test.go

```diff
@@ -97,7 +97,7 @@ func TestConvertToYARPCErrorApplicationErrorMeta(t *testing.T) {
 	require.NotNil(t, resw.ApplicationErrorMeta)
 	assert.Equal(t, "StringValue", resw.ApplicationErrorMeta.Name, "expected first error detail name")
 	assert.Equal(t,
-		`[]{ StringValue{value:"detail message" } , Int32Value{value:42 } , BytesValue{value:"detail bytes" } }`,
+		`[]{ StringValue , Int32Value , BytesValue }`,
```

Comment:
expected values were part of the stringified version of the proto message. This was possible because golang/proto was used to expose `CompactTextString` API. With v2 this API has been deprecated https://github.com/protocolbuffers/protobuf-go/blob/master/internal/protolegacy/proto.go#L93. Hence not possible to print the entire message.